src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/ActionSvcClient.java

```diff
@@ -172,6 +166,27 @@ public ActionPlanDTO getActionPlanBySelectorsSocial(String collectionExerciseId)
     return actionPlans.iterator().next();
   }
 
+  public ActionPlanDTO getActionPlanById(final UUID actionPlanId) {
```

Comment:
I cant see where this is used. whats calling this?

Reply:
removed the unused client method.